tools/lib/h5diff.c

```diff
@@ -91,35 +97,6 @@ phdiff_dismiss_workers(void)
     for (i = 1; i < g_nTasks; i++)
         MPI_Send(NULL, 0, MPI_BYTE, i, MPI_TAG_END, MPI_COMM_WORLD);
 }
-
-/*-------------------------------------------------------------------------
- * Function: print_incoming_data
- *
- * Purpose:  special function that prints any output that has been sent to the manager
- *           and is currently sitting in the incoming message queue
- *
- * Return:   none
- *-------------------------------------------------------------------------
- */
-
-static void
-print_incoming_data(void)
```

Comment:
These changes have refactored things such that there is only one occurrence where printed output is dealt with, so this function isn't really needed anymore. It also had the potential for ending up in an in-between state of printing output depending on the ordering of incoming messages from different worker tasks.